### PR TITLE
Fix documentation link

### LINKED ISSUE #25727: [Documentation]: Essentials Actions references @storybook/action-addons both in the current documentation and in the advanced/legacy documentation
### Describe the problem

Essentials Actions documentation references @storybook/action-addons both in the [current documentation](https://storybook.js.org/docs/essentials/actions#action) and in the [advanced/legacy documentation](https://github.com/storybookjs/storybook/blob/master/addons/actions/ADVANCED.md#manually-specified-actions).

Should we stop using the action function exported from @storybook/action-addons? And, will the action function from @storybook/action-addons be deprecated in v8 of storybook like the advanced/legacy documentation seems to suggest?

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by peschee**
Closes #25727

## What I did

Fixed the link to the `ADVANCED.md` file in the action addon section.

#### The changes in this PR are covered in the following automated tests:
- [ ] stories
- [ ] unit tests
- [ ] integration tests
- [ ] end-to-end tests

#### Manual testing

_This section is mandatory for all contributions. If you believe no manual test is necessary, please state so explicitly. Thanks!_

### Documentation

- [x] Add or update documentation reflecting your changes
- [ ] If you are deprecating/removing a feature, make sure to update
      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

## Checklist for Maintainers

- [ ] When this PR is ready for testing, make sure to add `ci:normal`, `ci:merged` or `ci:daily` GH label to it to run a specific set of sandboxes. The particular set of sandboxes can be found in `code/lib/cli/src/sandbox-templates.ts`
- [ ] Make sure this PR contains **one** of the labels below:
   <details>
     <summary>Available labels</summary>

     - `bug`: Internal changes that fixes incorrect behavior.
     - `maintenance`: User-facing maintenance tasks.
     - `dependencies`: Upgrading (sometimes downgrading) dependencies.
     - `build`: Internal-facing build tooling & test updates. Will not show up in release changelog.
     - `cleanup`: Minor cleanup style change. Will not show up in release changelog.
     - `documentation`: Documentation **only** changes. Will not show up in release changelog.
     - `feature request`: Introducing a new feature.
     - `BREAKING CHANGE`: Changes that break compatibility in some way with current major version.
     - `other`: Changes that don't fit in the above categories.
   
   </details>

### 🦋 Canary release

This PR does not have a canary release associated. You can request a canary release of this pull request by mentioning the `@storybookjs/core` team here.

_core team members can create a canary release [here](https://github.com/storybookjs/storybook/actions/workflows/canary-release-pr.yml) or locally with `gh workflow run --repo storybookjs/storybook canary-release-pr.yml --field pr=<PR_NUMBER>`_

## Greptile Summary

This PR fixes a documentation link in the Actions addon section of the Storybook documentation.

- Updated link to ADVANCED.md in `docs/essentials/actions.mdx` for improved accuracy
- Corrected path to `../../../code/addons/actions/ADVANCED.md` for advanced usage information
- Enhances user experience by providing easier access to legacy usage documentation

### CODE REVIEW

**Comment by peschee** [comment]
I wasn't able to add the `documentation` label myself…

File: docs/essentials/actions.mdx [hunk 1]
```diff
@@ -115,4 +115,4 @@ Allows you to create an action that appears in the actions panel of the Storyboo
 
 ## Advanced / legacy usage
 
-There are also some older ways to use actions as documented in the [advanced README](../../addons/actions/ADVANCED.md).
+There are also some older ways to use actions as documented in the [advanced README](../../../code/addons/actions/ADVANCED.md).
```
Comment (jonniebigodes) [inline docs/essentials/actions.mdx:118]: @peschee, when you have a moment, can you adjust the link to be as follows:
```suggestion
There are also some older ways to use actions as documented in the [advanced README](https://github.com/storybookjs/storybook/blob/next/code/addons/actions/ADVANCED.md).
```

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@peschee thank you for taking the time to put together this pull request and helping us improve the documentation by catching this small but rather important issue with this link. We appreciate it 🙏 !

I left a small item for you to look into when you have a moment.

Let me know and we'll go from there.

Have a great day.

Stay safe

**Comment by jonniebigodes** [comment]
@peschee, thank you for putting together this pull request and helping us improve the documentation. We appreciate it 🙏 ! However, the changes you're introducing have already been added to a recent pull request. I hope you don't take it the wrong way, and I look forward to your next contribution.

Have a great day.

Stay safe.

**Comment by peschee** [comment]
All good @jonniebigodes ;)

**Pull request closed without merging**